dev/devicelab/lib/tasks/microbenchmarks.dart

```diff
@@ -115,7 +115,7 @@ Future<Map<String, double>> _readJsonResults(Process process) {
   final StreamSubscription<String> stdoutSub = process.stdout
       .transform(const Utf8Decoder())
       .transform(const LineSplitter())
-      .listen((String line) {
+      .listen((String line) async {
```

Comment:
`async` should not be necessary here?

Reply:
Whoops, that was from when I added some sleeps in here testing the process killing. I noticed it and removed it but you were quicker than me! :)